source/about-mongosync.txt

```diff
@@ -157,6 +157,73 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Consider the following when using ``mongosync`` for continuous synchronization.
+
+Consistency 
+~~~~~~~~~~~
+
+For the destination cluster to have an up to date, consistent view of the 
+source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
+If the source cluster shuts down before ``mongosync`` can commit, such as in 
+a disaster scenario, the destination cluster might not have a consistent 
+snapshot of the source data, and transactions might not appear atomically on 
+the destination cluster. This can occur because ``mongosync`` temporarily alters 
+some collection characteristics during synchronization. The original values are 
+restored during the commit process.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Change
+     - Description
+  
+   * - Unique Indexes
+     - Unique indexes on the source cluster synchronize as non-unique indexes 
+       on the destination cluster.
+
+   * - TTL Indexes
+     - Synchronization sets ``expireAfterSeconds`` to the value of ``MAX_INT``
+       on the destination cluster.
+
+   * - Hidden Indexes
+     - Synchronization replicates hidden indexes as non-hidden.
+
+   * - Write Blocking
+     - If you enable write-blocking, ``mongosync`` blocks writes: 
+
+       - On the destination cluster during sync.
+       - On the source cluster when ``commit`` is received.
+
+   * - Capped Collections
+     - Synchronization sets capped collections to the maximum allowable
+       size. 
+
+   * - Dummy Indexes 
+     - In some cases, synchronization may create dummy indexes on the 
+       destination to support writes on sharded or collated collections. 
+  
+For any continuous synchronization use cases with ``mongosync``, ensure that 
+``mongosync`` commits before cutting over from the source to the 
+destination.
+
+.. note:: 
+  
+   After commit, you can't resume continuous sync between two clusters since 
+   ``mongosync`` can only sync into empty destination clusters. To start a 
+   new continuous sync operation, use a new empty destination cluster.
```

Comment:
CC @rohshar we may also want to add a note on reverse here.

Reply:
You can also use https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/api/reverse/ if you'd like to reverse the direction of the sync. This will ensure that any writes that occurred on the destination after cutover are replicated to the original source cluster.

Reply:
^ at second look, adding the note on filtered sync / reverse limitations will likely be out of scope for this note, but by linking the reverse endpoint, I think we've appropriately covered our bases and left a trail for users to follow to see those limiations.